lib/src/authentication/sign_in/sign_in_view_model.dart

```diff
@@ -0,0 +1,46 @@
+import "package:flutter/material.dart";
+import "package:go_router/go_router.dart";
+import "package:kana_to_kanji/src/glossary/glossary_view.dart";
+import "package:stacked/stacked.dart";
+
+class SignInViewModel extends FutureViewModel {
```

Comment:
There is no need here for a `FutureViewModel` a `BaseViewModel` should be enough, except if I missed something?

Reply:
En effet, c'est à corriger

Reply:
corrigé